data/campaigns/Eastern_Invasion/scenarios/10_Dark_Sanctuary.cfg

```diff
@@ -0,0 +1,2776 @@
+        [/delay]
+
+        [sound]
+            name=explosion.ogg
+        [/sound]
+
+        {COLOR_ADJUST 265 265 265}
+        {REPLACE_SCENARIO_MUSIC knalgan_theme.ogg}
+
+        {MODIFY_TERRAIN Uh^Dr 8 7}
+        {MODIFY_TERRAIN Uh^Dr 9 7}
+        {MODIFY_TERRAIN Ur^Dr 10 7}
+        {MODIFY_TERRAIN Ur^Dr 10 8}
+        [redraw]
+            clear_shroud=yes
+            side=1
+        [/redraw]
+
+        {NAMED_UNIT 2 (Orcish Sovereign) 9 7 "Chief Dra-Nak" (_"Chief Dra-Nak") ()} {FACING ne}
+        [+unit]
+            profile=portraits/orcs/ruler-2.webp
+            canrecruit=yes
+            [modifications]
+                {TRAIT_INTELLIGENT}
+                {TRAIT_STRONG}
+            [/modifications]
+        [/unit]
+        {NAMED_UNIT 5 (Goblin Spearman) 10 7 "Cowardly Goblin" (_"Cowardly Goblin") ()} {FACING ne}
+        {GENERIC_UNIT 2 (Orcish Warrior) 9 8}
+        {GENERIC_UNIT 2 (Orcish Warrior) 12 7}
+        {GENERIC_UNIT 2 (Orcish Warlord) 10 9
+        {GENERIC_UNIT 2 (Orcish Slurbow) 10 8}}
+        {GENERIC_UNIT 2 (Orcish Crossbowman) 9 9}
+        {GENERIC_UNIT 2 (Orcish Crossbowman) 7 10}
+        {GENERIC_UNIT 2 (Goblin Knight) 7 8}
+        {GENERIC_UNIT 2 (Goblin Pillager) 6 9}
+
+        {GENERIC_UNIT 2 (Orcish Warrior) 12 3}
+        {GENERIC_UNIT 2 (Orcish Crossbowman) 14 6}
+        {GENERIC_UNIT 2 (Orcish Slurbow) 14 5}
+        {GENERIC_UNIT 2 (Orcish Warlord) 10 5}
+
+        [delay]
+            time=300
+        [/delay]
+
+        {COLOR_ADJUST 0 0 0}
+
+        [message]
+            speaker="Chief Dra-Nak"
+            #po: again, implying that Whitefang has other chiefs, but that Dra-Nak is particularly powerful
+            message= _ "Hah, did you think you were safe hiding back in here? Nobody can escape from Dra-Nak, greatest chief east of the Listra."
+        [/message]
+        [message]
+            speaker="Cowardly Goblin"
+            message= _ "See, see! I told you there were humans in here! And I showed you the secret passage! Reward, reward?"
+        [/message]
+        [message]
+            speaker="Chief Dra-Nak"
+            message= _ "Sure, here you go. You did well."
+        [/message]
+        [sound]
+            name=gold.ogg
+        [/sound]
+
+        {MOVE_UNIT (id="Cowardly Goblin") 1 13}
+
+        [message]
+            speaker="Chief Dra-Nak"
+            message= _ "As for you humans, you might as well convince your friends outside to surrender. Otherwise, we’ll slaughter you all starting with that injured old man. I gave you a chance to do things fairly before, but now we’re doing things my way."
+        [/message]
+        [message]
+            speaker="Gweddry"
+            message= _ "<span size='small'><i>I don’t know what these orcs want with us, but we can’t hope to escape with Dacyn so badly hurt.</i></span>"
+        [/message]
+        [message]
+            speaker="Gweddry"
+            message= _ "Very well orc, we will all surrender."
```

Comment:
'we will surrender' is sufficient.

Reply:
"we will all" is a little clumsy, but I wanted to include Owaec's forces outside in the surrender.  There may be a better way to phrase things.  Note that Gweddry and Owaec aren't in communication at this moment.

Reply:
To me, as a native Anglophone, the implication is clear with just 'we surrender'. But I suppose it might not necessarily be so for non-native readers.

Reply:
"we all surrender" could also work (i.e., strike the "will" instead of the "all")